Zilon.Core/Zilon.Core/Persons/Survival/DiseaseEffect.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+using Zilon.Core.Components;
+
+namespace Zilon.Core.Persons.Survival
+{
+    public class DiseaseEffect : IPersonEffect
+    {
+        public DiseaseEffect(IDisease disease)
+        {
+            Disease = disease;
+        }
+
+        public IDisease Disease { get; }
+
+        public event EventHandler Changed;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'Changed' or invoke it.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5271000)